app/models/grda_warehouse/tasks/identify_duplicates.rb

```diff
@@ -120,10 +120,18 @@ def merge_history
 
     def find_merge_candidates
       to_merge = Set.new
+      all_splits = GrdaWarehouse::ClientSplitHistory.pluck(:split_from, :split_into)
+      splits_by_from = all_splits.group_by(&:first)
+      splits_by_into = all_splits.group_by(&:last)
```

Comment:
Thinking through this, I'm pretty sure this needs to look at the source clients at some level.  (It may have to look for the destinations of the original sources) but the destination client ids will change over time as source clients are merged and split.  The source client ids should be pretty persistent.  Hopefully a test suite will cover this.